egov-dss-dashboards/dashboard-analytics/MasterDashboardConfig.json

```diff
@@ -114,8 +177,71 @@
       "visualizations": [
         {
           "row": 1,
-          "name": "DCC_DISTRICT_HEALTH",
-          "vizType": []
+          "name": "DSS_HEALTH_OVERVIEW",
+          "vizArray": [
+            {
+              "id": 302,
+              "name": "DSS_HEALTH_POPULATION_COVERED",
+              "label": "DSS_HEALTH_POPULATION_COVERED",
+              "vizType": "stacked-collection",
+              "noUnit": false,
+              "isCollapsible": false,
+              "charts": [
+                {
+                  "id": "provincialOrDistrictPopulationCovered",
+                  "name": "DSS_HEALTH_PROVINCIAL_DISTRICT_POPULATION_COVERED",
+                  "code": "",
+                  "chartType": "metric",
+                  "filter": "",
+                  "headers": []
+                },
+                {
+                  "id": "provincialOrDistrictTotalPopulationCovered",
```

Comment:
rename this appropriately once neel confirms that "id" can have "overview" instead of "provincialOrDistrict"

Reply:
Cant you use the same chart of total population covered created for national level?